testapps/integration/web/index.php

```diff
@@ -29,12 +29,12 @@
 });
 
 $app->post('/logging_standard', function (Request $request) {
-    $logName = $request->get('log_name');
-    $token = $request->get('token');
+    $logName = $request->request->get('log_name');
```

Comment:
Yeah the README says so, but the code actually adds log_name field.
https://github.com/GoogleCloudPlatform/runtimes-common/blob/master/integration_tests/testsuite/test_logging_standard.py#L35
and
https://github.com/GoogleCloudPlatform/runtimes-common/blob/master/integration_tests/testsuite/test_util.py#L74

Adding @nkubala to confirm